packages/salesforcedx-vscode-core/src/telemetry/telemetryReporter.ts

```diff
@@ -112,6 +112,16 @@ export default class TelemetryReporter extends vscode.Disposable {
       /^(\d+)(\.\d+)?(\.\d+)?(.*)/,
       '$1$2$3'
     );
+    const cpus = os.cpus();
```

Comment:
![screen shot 2019-03-04 at 1 26 57 pm](https://user-images.githubusercontent.com/1639375/53761154-16c21700-3e82-11e9-9fec-6fff2bbdbfa6.png)
This is what the newly added fields look like in AppInsights.

The information about CPUs and total memory is fetched once when telemetry is first initialized on extension activation. It is also possible to get information on the amount of *available* memory from the `os` package, but it seems like that information would be most useful if we fetched it each time of each telemetry event so that we can get a snapshot of available memory as the commands are being executed. @lcampos  Is this information that I should add to each event?

Reply:
I think this is ok. The telemetryReporter.ts is supposed to be a temporary fork of https://github.com/Microsoft/vscode-extension-telemetry since we wanted to add some capabilities to it and were waiting on those changes to get merged. Once we merge this we should open an issue on the repo and open a PR to see if we can get them merged.

Reply:
For the available memory info, I agree that it makes more sense to be captured at a per command basis. Lets handle it as a separate work item.